pages/api/storeNlpData.ts

```diff
@@ -0,0 +1,38 @@
+import { MongoClient } from "mongodb"
+import { NextApiRequest, NextApiResponse } from "next"
+import redis from "@/utils/redis"
+import * as keys from "@/utils/keys"
+import { NLP_KEY } from "@/constants"
+
+export default async function handler(_req: NextApiRequest, res: NextApiResponse) {
+  let numberOfDocuments = 0
+  const date = new Date().toLocaleDateString()
+  await MongoClient.connect(
+    `mongodb://${keys.mongoUser}:${keys.mongoPassword}@${keys.serverHost}:${keys.mongoPort}`,
+    {}
+  ).then(async (client) => {
+    await redis.keys(`${NLP_KEY}*`).then(
```

Comment:
this runs every night. the first thing it does is get a list of all the keys in redis that start with nlp. this will give us every dashboard that was created that day. ie. [nlp/user1/30days, nlp/user1/60days, nlp/user2/30days]